packages/platform/src/__tests__/platform.test.ts

```diff
@@ -128,10 +128,42 @@ describe('platform', () => {
 
     platform.loadMetadata(ids.meta, {
       M1: 'hey',
-      M2: 'there'
+      M2: 'there',
     })
 
     expect(platform.getMetadata(ids.meta.M1)).toBe('hey')
     expect(platform.getMetadata(ids.meta.M2)).toBe('there')
   })
+
+  it('should set metadata', () => {
+
+    const ids = identify('test' as AnyPlugin, {
+      meta: {
+        M1: 'test-meta' as Metadata<string>,
+        M2: 'test-meta2' as Metadata<string>,
+        M3: '' as Metadata<string>,
+      }
+    })
+    platform.setMetadata(ids.meta.M1, 'new-test-meta')
+    expect(platform.getMetadata(ids.meta.M1)).toBe('new-test-meta')
+    platform.setMetadata(ids.meta.M2, undefined)
+    expect(platform.getMetadata(ids.meta.M2)).toBe(undefined)
+    platform.setMetadata(ids.meta.M3, null)
+    expect(platform.getMetadata(ids.meta.M3)).toBe(null)
+  })
+
+  it('should get plugins infos', () => {
+
+    const ids = identify('test' as AnyPlugin, {
+      meta: {
+        M: '' as Metadata<string>
+      }
+    })
+
+    platform.loadMetadata(ids.meta, {
+      M: 'hey'
+    })
+    expect(platform.peekResource(ids.meta.M)).toBe('hey')
+  })
```

Comment:
This is not related to plugin infos. Maybe it's better to check if `Metadata` also accessible as `Resource` and add 
```typescript
expect(platform.peekResource(ids.meta.M)).toBe('hey')
```

to `should set metadata` test above?